### PR TITLE
My code review suggestions.

### PR DESCRIPTION
**Pull request by phanga**
Description/Analysis of the Project
This is an app that identifies different recycling places and can add them to favorites.

Was the program available in UC Github on time?
Yes

Is the program documented/commented well enough for you to understand?
Yes

Does the program compile?
Yes

Rationale Behind Changes
I began to add in a potential addition to the project and added a link on how to integrate it. I also fixed a minor typo based on best practices and modified a certain text field for functionality purposes.

3 Commits

Added config.yml
https://github.com/phanga/MyPerishablePlannerV21001/commit/cf1ba214a998b5f71689497d85501e41fddd021a
Added colors and commented out class in DTO for project to build
https://github.com/phanga/MyPerishablePlannerV21001/commit/0a70725b9d4be9a96b57aeb0c9392961ab41c32f
Added start of AutoCompleteTextView
https://github.com/phanga/MyPerishablePlannerV21001/commit/cdafa67d4caddb3fbb0d50af98d6fbd18726904a

Things I Learned

How to add in AutoCompleteTextView
Using modifiers inside certain Columns and in Text Fields
Practicing and modifying best practices

### CODE REVIEW

File: app/src/main/java/com/findmyrecycling/MainActivity.kt [hunk 1]
```diff
@@ -35,15 +35,15 @@ fun RecycleSearch(name: String) {
     var location by remember{ mutableStateOf("")}
 
     Column {
-        OutlinedTextField(
+        TextField(
```
Comment (phanga) [inline app/src/main/java/com/findmyrecycling/MainActivity.kt:38]: Made changes to this based on what I found for functionality and looks. https://uxmovement.com/forms/why-you-shouldnt-use-solid-or-underlined-text-fields/

File: app/src/test/java/com/findmyrecycling/ProductTests.kt [hunk 2]
```diff
@@ -39,7 +39,7 @@ class ProductTests {
     }
 
     @After
-    fun TearDown(){
+    fun tearDown(){
```
Comment (phanga) [inline app/src/test/java/com/findmyrecycling/ProductTests.kt:42]: Change made just based on best practices with Camel Case.

File: app/build.gradle [hunk 3]
```diff
@@ -34,6 +34,7 @@ android {
         useIR = true
     }
     buildFeatures {
+        viewBinding true
```
Comment (phanga) [inline app/build.gradle:37]: I started to add the beginnings of one of the projects I saw you all were going to implement which is the AutoCompleteTextview. The link to the tutorial on how to add this in is here --> https://tutorialwing.com/android-autocompletetextview-using-kotlin-with-example/

Comment (discospiff) [reply app/build.gradle:37]: What purpose does this serve?
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

File: app/src/main/java/com/findmyrecycling/MainActivity.kt (hunk 1 above)
Comment (discospiff) [reply app/src/main/java/com/findmyrecycling/MainActivity.kt:38]: Duplicates a previous review.  When that happens, first in wins.

File: app/src/main/java/com/findmyrecycling/MainActivity.kt [hunk 4]
```diff
@@ -35,15 +35,15 @@ fun RecycleSearch(name: String) {
...
             value = recyclable,
             onValueChange = {recyclable = it},
             label = { Text(stringResource(R.string.recyclable)) },
             modifier = Modifier.fillMaxWidth()
         )
 
 
-        OutlinedTextField(
+        TextField(
```
Comment (discospiff) [inline app/src/main/java/com/findmyrecycling/MainActivity.kt:46]: Duplicates a previous review.  When that happens, first in wins.

File: app/src/test/java/com/findmyrecycling/ProductTests.kt (hunk 2 above)
Comment (discospiff) [reply app/src/test/java/com/findmyrecycling/ProductTests.kt:42]: Duplicates a previous review.  When that happens, first in wins.

**Comment by discospiff** [comment]
No need to merge this branch.

Most of the changes here duplicate what can be found in a previous review.

File: app/build.gradle (hunk 3 above)
Comment (phanga) [reply app/build.gradle:37]: @discospiff My idea behind this was to send a resource for the team to begin to look at and implement the start of that resource.

**Pull request closed without merging**